nodes/test/conftest.py

```
@@ -52,20 +53,24 @@
 # Test Configuration
 # =============================================================================
 
+
 class TestConfig:
     """Test configuration loaded from environment variables."""
-    
+
     def __init__(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)